src/test/java/com/epam/mjc/collections/list/ArrayListCreatorTest.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.mjc.collections.list;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class ArrayListCreatorTest {
+
+    @Test
+    public void createArrayListFromEmptyList() {
+        ArrayListCreator arrayListCreator = new ArrayListCreator();
+        ArrayList<String> expectedArrayList = new ArrayList<>();
+
+        ArrayList<String> actualArrayList = arrayListCreator.createString(new ArrayList<>());
+        assertEquals(expectedArrayList, actualArrayList);
+    }
+
+    @Test
+    public void createArrayListFromListWithTwoElements() {
+        ArrayListCreator arrayListCreator = new ArrayListCreator();
+        ArrayList<String> expectedArrayList = new ArrayList<>();
+
+        ArrayList<String> actualArrayList = arrayListCreator.createString(List.of("Hello", "Word"));
+        assertEquals(expectedArrayList, actualArrayList);
+    }
+
+    @Test
+    public void createArrayListFromListWithThreeElements() {
+        ArrayListCreator arrayListCreator = new ArrayListCreator();
+        ArrayList<String> expectedArrayList = new ArrayList<>();
```

Comment:
Could you use `List.of(...)`? For example:
`new ArrayList(List.of("you", "you"))`